update_server.xml

```diff
@@ -0,0 +1,25 @@
+<updates>
+    <update>
+        <name>JEM - Joomla Event Manager</name>
+        <description>
+            JEM - an Event Manager for Joomla
+        </description>
+        <element>com_jem</element>
+        <type>component</type>
+        <version>2.2.0</version>
+        <infourl title="JED - Joomla Event Manager">
+            http://www.joomlaeventmanager.net
+        </infourl>
+        <downloads>
+            <downloadurl type="full" format="zip">
+                https://github.com/jemproject/JEM-Project/archive/v2.2.0-stable.zip
```

Comment:
#77 added update-implementation for Joomla 3.
`<targetplatform name="joomla" version="3.*"/>`